Gopkg.lock

```diff
@@ -814,6 +814,7 @@
     "k8s.io/apimachinery/pkg/apis/meta/v1",
     "k8s.io/apimachinery/pkg/runtime",
     "k8s.io/apimachinery/pkg/runtime/schema",
+    "k8s.io/apimachinery/pkg/util/clock",
```

Comment:
Does this need to be vendored in? 
Also note that this commit should come before the commit where `clock` is being used.

Reply:
Then the update to Gopkg.lock should be in the commit where it was vendored in.

Reply:
The `util` package was vendored in on day 1, I believe. Probably as a result of `operator-sdk init`. Now that `/util/clock` is being used `dep` has added it to its `imports` list.

Reply:
I would suggest moving this to the commit where you start using `/util/clock` then and add an explanation in the commit message

Reply:
moved back into original commit.